js/form-validation.js

```diff
@@ -0,0 +1,61 @@
+function showErrorMsg(inputElem, message) {
+  const msg = inputElem.parentNode.querySelector('lblErrorMsg');
+  if (message.trim() === '') {
+    message = 'Sending your message...';
+    msg.style.color = 'green';
+  }
+
+  msg.innerText = message;
+}
+
+function IsEmpty(inputElem) {
+  return (inputElem.value.trim() === '');
+}
+
+function invalidEmail(inputElem) {
+  // validate email format
+  inputElem.value = inputElem.value.toLowerCase();
+
+  const emailRegex = /^(([^<>()\]\\.,;:\s@"]+(\.[^<>()\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+  const email = inputElem.value.trim();
+  return !(emailRegex.test(email));
+}
+
+const form = document.getElementById('contact-form');
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+
+  let errMsg = '';
+
+  if (IsEmpty(form.elements.full_name)) {
+    errMsg += 'name, ';
+    form.elements.full_name.focus();
+  }
+  if (IsEmpty(form.elements.email)) {
+    if (errMsg.trim() === '') {
+      form.elements.email.focus();
+    }
+    errMsg += 'email, ';
```

Comment:
- [ ] Great job working on the validation so far but I think you forget to implement an error message when the user add an email with a capital letter when I tried to enter an email with capital letter the form just submit sucessfully I noticed you are transforming it to be in small letters then you let the form submit 🙏🏻😊 that's good but also you should show up the error message to let the user edit it with himself 😊🙏🏻🚀💯

## An error message should show up to the user near the submit button informing them of the error and the form is not sent.✅✅✅✅

https://user-images.githubusercontent.com/58553711/233231056-3cecca4a-d31f-49ef-b71d-a29c8fdc4f33.mp4

Reply:
Thank you @Reem-lab . I did it in that way because I thought it was a better UX, if the user don't recieve and error and instead of it, the app solve it for him.
But  I'l do the correction. 👍